.travis.yml

```diff
@@ -20,9 +20,9 @@ env:
   - DH_REPO="cloudsuite/java" IMG_TAG="openjdk8" DF_PATH="./commons/java/openjdk8"
 
   # Hadoop
-  - DH_REPO="cloudsuite/hadoop" IMG_TAG="latest" DF_PATH="./commons/hadoop/2.9.1"
-  - DH_REPO="cloudsuite/hadoop" IMG_TAG="2.7.6" DF_PATH="./commons/hadoop/2.7.6"
-  - DH_REPO="cloudsuite/hadoop" IMG_TAG="2.9.1" DF_PATH="./commons/hadoop/2.9.1"
+  # - DH_REPO="cloudsuite/hadoop" IMG_TAG="latest" DF_PATH="./commons/hadoop/2.9.1"
```

Comment:
These should not be modified.
Also please pull all the recent changes made to the dev branch into your branch so we won't have a conflict.

Reply:
@alok123t thanks for the recent update but there is still a problem with your branch. did you use rebase to bring all the changes to your branch? or something else?
Because when I go to the tab where the changes are shown I see that a lot of changes there were supposed to be in the dev branch already.
It's like your branch hasn't branched off of the new dev. You need to use rebase to bring all the changes into your branch in a way that your branch has started off of the newly updated dev.
Otherwise, you have to create a new PR just with your changes.

Reply:
I think this might be helpful to look at: https://github.com/edx/edx-platform/wiki/How-to-Rebase-a-Pull-Request

Reply:
Updated the PR after rebasing